source/NVDAObjects/window/scintilla.py

```diff
@@ -193,9 +194,9 @@ def _getLineOffsets(self,offset):
 			# Lines in Scintilla refer to document lines, not wrapped lines.
 			# There's no way to retrieve wrapped lines, so use screen coordinates.
 			y=watchdog.cancellableSendMessage(self.obj.windowHandle,SCI_POINTYFROMPOSITION,None,offset)
-			top,left,width,height=self.obj.location
-			start = self._getOffsetFromPoint(0,y)
-			end=self._getOffsetFromPoint(width,y)
+			location=self.obj.location
+			start = self._getOffsetFromPoint(location.left, location.top+y)
+			end=self._getOffsetFromPoint(location.right, location.top+y)

```

Comment:
When using _getPointFromOffset above, this becomes:

```suggestion
			end=self._getOffsetFromPoint(location.right, y)
```

Reply:
Hi Leonard,

Yes it makes sense. I initially didn't write it like this because I thought a useless call to `watchdog.cancellableSendMessage` to get the X coordinate wouldn't be accepted. But I don't really know how expensive is that type of call.